src/GSoft.AspNetCore.Authentication.ClientCredentialsGrant.Tests/AuthenticationBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace GSoft.AspNetCore.Authentication.ClientCredentialsGrant.Tests;
+
+public class AuthenticationBuilderExtensionsTests
+{
+    [Fact]
+    public async Task GivenAnAuthenticationBuilder_WhenConfigsArePresent_ThenOptionsAreSet()
+    {
+        var inMemorySettings = new Dictionary<string, string>
+        {
+            { $"{ClientCredentialsDefaults.ClientCredentialsConfigSection}:Authority", "https://identity.local" },
+            { $"{ClientCredentialsDefaults.ClientCredentialsConfigSection}:Audience", "audience" },
+        };
+        
+        var services = new ServiceCollection();
+        var configuration = new ConfigurationBuilder().AddInMemoryCollection(inMemorySettings).Build();
+        services.AddSingleton<IConfiguration>(configuration);
+        
+        var authenticationBuilder = new AuthenticationBuilder(services);
+        authenticationBuilder.AddClientCredentials();
```

Comment:
`services.AddAuthentication().AddClientCredentials(...)` is smaller and actually more accurate about how to register the authentication services (affects the other tests too).